articles/143_legacy_interface_definition.md

```diff
@@ -39,21 +39,52 @@ Each field is described by a *type* and a *name*.
 A single data structure is called *message*.
 Each message has a *name*.
 Together with the name of the *package* a message can be uniquely identified.
+A message has the URN: package/msg/name`.
 
 ### Services
 
 For request / reply style communication the two exchanged data structures are related.
 These pairs of data structures are called *services*.
-A service is identified by its *name* and the *package* it is in.
 Each service describes two messages, one for the request data structure, one for the reply data structure.
+Together with the name of the *package* a
```

Comment:
This seems to end in the middle of the sentence?

Reply:
Yeah, this is vestigial. Removed in 30a99ff